kernel/entries.ml

```diff
@@ -40,7 +40,8 @@ type one_inductive_entry = {
   mind_entry_typename : Id.t;
   mind_entry_arity : constr;
   mind_entry_consnames : Id.t list;
-  mind_entry_lc : constr list }
+  mind_entry_lc : constr list;
+}
```

Comment:
What about these unrelated cosmetic changes? (I prefer the new variant but it's a potential flamewar.)

Reply:
They are related in the sense of applying a `consistency of style`. I'm not saying that one is best than the other but in `entries.ml` at least, there is already a majority of single `}`. (Would it be in `declare.ml`, consistency of style may give instead a majority of `; field : type` style...)

Reply:
As far as I am concerned, I hate the `; field : type` style with a passion. The semicolon is semantically related to the preceding field, not the rhs one.